docs/Installing.asciidoc

```diff
@@ -120,6 +160,8 @@ openQA runs on each boot, you should also +systemctl enable+ the same services.
 
 [source,sh]
 --------------------------------------------------------------------------------
+systemctl enable postgresql
+systemctl enable openqa-resource-allocator
```

Comment:
the openqa-webui.service requires  openqa-resource-allocator.service openqa-scheduler.service openqa-websockets.service - as such they don't need manual starting.